pkg/tests/end_to_end_tests/create_test.go

```diff
@@ -1573,6 +1573,121 @@ func TestConfigMaintenanceJobs(t *testing.T) {
 	})
 }
 
+func TestRegisterMetricView(t *testing.T) {
```

Comment:
do we have any testing for continous aggregate views?

Reply:
I didn't think it was necessary tbh. since in essence they behave the same way as plain views.

Reply:
yes but since this is the main goal of this thrust of work we should have at least some end_to_end tests using that to make sure the integration with timescale works as expected. I would actually even start a new test go file. `continuous_agg_test.go`